summary_workflows/identification/specification/example_files/aggregation_out.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Can you make this a list of 2 aggregation objects, and add a second object with the plot for the 50nt metrics that are present in the assessment_out.json?